### ISSUE TITLE
[Feat] New loss functions

### ISSUE THREAD

**Issue by yehyunsuh** [yehyunsuh/Total-Knee-Replacement-Post-Surgical-Assessment#17]
## What
New loss functions
<br>

## Why
After giving weight to the label values, accuracy seemed to have better performance.
<img width="2672" alt="image" src="https://user-images.githubusercontent.com/73840274/214388455-040af9fb-f730-4439-b6cc-4691990205e8.png">
<br>

## How
- [x] Try other loss that balances dataset with unbalanced labels.

**Comment by yehyunsuh** [comment]
Loss function was not the problem. It was the learning rate. Close Issue.
